Content.Shared/Atmos/PipeDirection.cs

```diff
@@ -34,8 +34,10 @@ public enum PipeDirection
         //Half of a pipe in a direction
         North = 1 << 0,
         South = 1 << 1,
-        West = 1 << 2,
-        East = 1 << 3,
+        West  = 1 << 2,
+        East  = 1 << 3,
+        Up    = 1 << 4,
+        Down  = 1 << 5,
 
```

Comment:
As discussed on discord, these should be renamed to `Port`/`Connector` to indicate their *actual* use as a workaround for canister/connectors.

Reply:
Done